### ISSUE TITLE
Incorrect example value under Request Body

### ISSUE THREAD

**Issue by AndriyVasyk** [mczachurski/Swiftgger#21]
Hello!
I am experiencing problems with incorrect values I used to demonstrate params for request body. See attached image.
<img width="302" alt="Screenshot 2021-06-07 at 14 57 54" src="https://user-images.githubusercontent.com/24439761/121015082-45d33b80-c7a3-11eb-943d-4ea55fcaed98.png">

And see the code snippet and json below:
<code>
struct TestObject {
    var test: String
    var authClientIds: [UUID]
}
</code>
`
let openAPIBuilder = OpenAPIBuilder(
    title: "Tasker server API",
    version: "1.0.0",
    description: "This is a sample server for task server application."
)
.add([
    APIObject(object: TestObject(test: "Test-ID", authClientIds: [UUID(), UUID()])),
])
.add(APIController(name: "Users", description: "Controller where we can manage users", actions: [
    APIAction(method: .post,
              route: "/users",
              summary: "Test",
              description: "Test",
              request:
                APIRequest(object: TestObject.self, description: "Test description"),
              responses: [
                APIResponse(code: "200",
                            description: "Ok",
                            object: TestObject.self)
              ],
              authorization: false)
    ])
)

let openAPIDocument = openAPIBuilder.built()

let encoder = JSONEncoder()
encoder.outputFormatting = .prettyPrinted

let jsonData = try! encoder.encode(openAPIDocument)
let jsonString = String(bytes: jsonData, encoding: .utf8)
print(jsonString!)
return jsonString!
`
[json.txt](https://github.com/mczachurski/Swiftgger/files/6615323/json.txt)

What I am doing wrong?
I expect to get the following under Example Value:
<code>
{
  "authClientIds": ["26E490CF-31C5-4A12-A212-74BF7A896CC6", "39E490CF-31C5-4A12-A212-74BF7A896CC6"]
 "test": "test-str"
}
</code>
Could you please take a look at this issue?

**Comment by gulivero1773** [comment]
![Снимок экрана 2021-06-09 в 17 17 42](https://user-images.githubusercontent.com/61082418/121371974-9d5ddc80-c946-11eb-8176-62d52e5dd990.png)
@mczachurski  It need not String array ( "[UUID]" ), It needs array String ( ["UUID"] ). You wrap all objects in a string, how can you avoid this?

**Comment by mczachurski** [comment]
Hi. I've created new version (1.5.0) which should properly generate model (JSON) for `UUID` properties. Could you verify if it's ok now for you?

**Comment by AndriyVasyk** [comment]
Hi @mczachurski ! Thanks a lot!
The new version works fine for single UUID property. However there is still problem with array of UUIDs. Corresponding JSON does not contain 'example'.
I tested with "testSchemaWithUUIDProperyShouldBeTranslatedToOpenApiDocument, OpenAPISchemasBuilderTests.swift":

```
      "VehicleKeys" : {
        "type" : "object",
        "properties" : {
          "arrayIds" : {
            "items" : {
              "type" : "string",
              "format" : "uuid"
            }
          },
          "singleId" : {
            "type" : "string",
            "format" : "uuid",
            "example" : "E621E1F8-C36C-495A-93FC-0C247A3E6E5F"
          }
        },
        "required" : [
          "singleId",
          "arrayIds"
        ]
      }
```

**Comment by AndriyVasyk** [comment]
Note, that 'example' also absent for me if to change [UUID] to [String]:
```swift
class VehicleKeys {
...
var arrayIds: [UUID]
...
}
```
to
```swift
class VehicleKeys {
...
var arrayIds: [String]
...
}
```

**Comment by mczachurski** [comment]
Yes `example` value for arrays is missing functionality. I will try to implement this in the next release.

**Comment by mczachurski** [comment]
Fix since version 2.0.
